profiles/opendj/terraform/opendj/api-services.tf

```diff
@@ -0,0 +1,15 @@
+module "enable-services" {
+  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=services-0.1.2"
+  enable_flag = "1"
+  providers {
+    google.target = "google"
+  }
+  project     = "${var.google_project}"
+  services    = [
+    "stackdriver.googleapis.com",
+    "logging.googleapis.com",
+    "sql-component.googleapis.com",
```

Comment:
not sure if these api service list is really accurate for opendj.  don't think it uses sql nor pubsub.  but since api-services are project wide and I do know that FC does utilize these services its not a big deal that they are listed here.  Up to you if you wanted to be more explicit with regard to the opendj service.

Reply:
No I think we do want to try and keep only relevant API services in the profiles, even if they are project-wide and persistent. I will update the list, thanks!